src/Raven.Server/Documents/Sharding/ShardedDatabaseContext.Operations.cs

```diff
@@ -112,23 +123,20 @@ protected override void RaiseNotifications(OperationStatusChange change, Abstrac
             }
             catch
             {
-                if (operation.Killable)
-                    await operation.KillAsync(waitForCompletion: true, t);
+                if (token != null)
```

Comment:
why?

Reply:
in the code that I replaced we had `if (operation.Killable)` which resolves to `Token != null` https://github.com/ravendb/ravendb/blob/v6.0/src/Raven.Server/Documents/Operations/AbstractOperation.cs#L34

Reply:
I liked the `Killable`, but it doesn't matter :)